docs/02.API-REFERENCE.md

```diff
@@ -234,6 +234,15 @@ memory blocks but the performance may drop after the garbage collection.
 
 *New in version 2.0*.
 
+## jerry_backtrace_frame_types_t
+
+List of backtrace frame types returned by
+[jerry_backtrace_get_frame_type](#jerry_backtrace_get_frame_type).
+
+ - JERRY_BACKTRACE_JS_CODE - frame is created for running ECMAScript code
```

Comment:
* I was wondering if having a `JERRY_BACKTRACE_UNKNOWN` like entry would make sense? Which would indicate in the future that there was some kind of error/or the frame's type is not known (eg.: native/external, incorrect input).
* Would the `JERRY_BACKTRACE_FRAME_JS` be a better name? This way the "frame" as information is in the name. The "_CODE" does not provide extra info in the current variant.
* The comment could be: `indicates that the frame is created for a  function/method implemented in JavaScript`.

Reply:
There should never been an unknown frame, if we add more frame types for the engine, this api should be extended as well.